contracts/stabilizer/TribeReserveStabilizer.sol

```diff
@@ -72,6 +75,20 @@ contract TribeReserveStabilizer is ITribeReserveStabilizer, ReserveStabilizer {
         return valid && ratio.lessThanOrEqualTo(_collateralizationThreshold);
     }
 
+    /// @notice delay the opening of the TribeReserveStabilizer until oracle delay duration is met
+    function startOracleDelayCountdown() external override {
+        require(isCollateralizationBelowThreshold(), "TribeReserveStabilizer: Collateralization ratio above threshold");
+        require(!isTimeStarted(), "TribeReserveStabilizer: timer started");
```

Comment:
Lines 80 and 81 need to have the sad path tested in the unit tests.

Reply:
done